test/e2e/qat/qatplugin_dpdk.go

```diff
@@ -115,7 +115,7 @@ func describeQatDpdkPlugin() {
 		})
 
 		ginkgo.It("deploys a crypto pod (openssl) requesting QAT resources", func() {
-			runCpaSampleCode(f, "4", resourceName)
+			runCpaSampleCode(f, "1", resourceName)
```

Comment:
Would you mind using a variable like "symmetricTest" or similar in the call? `runCpaSampleCode(f, symmetricTest, resourceName)`
Also, can you please add a comment, for future us, which declares the test numbers, their explanations and whether they work on simulated HW?

Reply:
Ah, yeah, it looks good! :)
Let me change it!

Reply:
it can be const maybe?

Reply:
You mean "comment" in the code, right?

Reply:
> it can be const maybe?

Yep

> You mean "comment" in the code, right?

Yes, comment in the code.